src/OrbitGl/TrackManager.cpp

```diff
@@ -441,6 +445,24 @@ SystemMemoryTrack* TrackManager::CreateAndGetSystemMemoryTrack(
   return system_memory_track_.get();
 }
 
+CGroupAndProcessMemoryTrack* TrackManager::CreateAndGetCGroupAndProcessMemoryTrack(
+    const std::array<std::string, orbit_gl::kCGroupAndProcessMemoryTrackDimension>& series_names) {
+  if (cgroup_and_process_memory_track_ == nullptr) {
+    constexpr uint8_t kTrackValueDecimalDigits = 2;
+    const std::string kTrackValueLabelUnit = "MB";
```

Comment:
We should discuss whether we wanna write `MB` or `MiB`. I'm strongly in favor of the latter since we divide by 1024. :slightly_smiling_face:

Reply:
I'm rather neutral on the topic, but note that until now in all other places where memory units are involved (not many though) we use the `MB` version (see `GetDisplaySize`), so we also need to be consistent (one way or the other).

Reply:
Thanks for the suggestion. I will prepare a follow up PR to change to consistently use `KiB` and `MiB` as the size units.